CONVENTION.md

```diff
@@ -0,0 +1,2 @@
+# HTML conventions
+- HTML element ids that are solely used for debug purposes are named `"debug-..."`, e.g., `"debug-game-name"`
```

Comment:
CSS classes thar are solely used for testability/readability purposes are named ...